wrapper/src/test/java/integration/refactored/container/ConnectionStringHelper.java

```diff
@@ -109,6 +110,23 @@ public static String getProxyWrapperUrl() {
         TestEnvironment.getCurrent().getInfo().getProxyDatabaseInfo().getDefaultDbName());
   }
 
+  public static String getWrapperClusterEndpointUrl() {
+    if (StringUtils.isNullOrEmpty(TestEnvironment.getCurrent().getInfo().getDatabaseInfo().getClusterEndpoint())) {
+      throw new RuntimeException("Cluster Endpoint is not available in this test environment.");
```

Comment:
should this message be added to the messages.properties file and use Messages.get()

Reply:
It's a test suite so we can more loose in terms on keeping all messages in message file. Also, this message file is for the driver and I'd like to avoid polluting it with message from test suite.